apps/loader/loader.js

```diff
@@ -114,15 +114,23 @@ function handleDownload(id) {
   var fileName='';
   store.getSlide(id)
       .then((response) => {
-        return response[0]['location'];
+        if (response[0]['location']) {
+          return response[0]['location'];
+        }
```

Comment:
```suggestion
        } else {
           throw new Error('Slide not available');
        }
```

Reply:
I also added something like this before but,
This `else` might not even execute because the error will be thrown in the fetch itself. Correct me if I am wrong.